jwala-integration-test/src/test/java/com/cerner/jwala/ui/selenium/component/JwalaUi.java

```diff
@@ -213,6 +213,10 @@ public void clickComponentForUpload(String component) {
         webElement.click();
     }
 
+    public void waitUntilElementIsClickable(final By by, final long timeout) {
```

Comment:
So basically this is the same as clickWhenReady except that it does not check for the busy screen and it accepts a timeout value. I suggest that we just "overload" clickWhenReady and write a java doc to describe to users the differences of the overloaded methods. In that way the code is easier to maintain since methods with the same "general" functionality have the same name. 

Example:
```java
   /**
    * Java Doc here
    */
    public void clickWhenReady(final By by) { ... }

   /**
    * Java Doc here
    */
    public void clickWhenReady(final By by, final long timeout) { ... }
```